specutils/tests/test_loaders.py

```diff
@@ -1273,3 +1273,85 @@ def test_galah_guess(remote_data_path):
         assert isinstance(spectra[1].uncertainty, StdDevUncertainty)
         assert spectra[1].meta.get("label") is not None
         assert spectra[1].meta.get("header") is not None
+
+
+# We cannot use remote_access directly in the MIRI MRS tests, because
```

Comment:
I am confused. Comment says you cannot use `remote_access` directly, but the code uses it directly. What is going on here?